namespaces/cloud-platform-live-0.k8s.integration.dsd.io/laa-apply-for-legalaid-uat/resources/main.tf

```diff
@@ -0,0 +1,54 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-1"
+}
+
+/*
+ * When using this module through the cloud-platform-environments, the following
+ * two variables are automatically supplied by the pipeline.
+ *
+ */
+
+variable "cluster_name" {}
+
+variable "cluster_state_bucket" {}
+
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "apply-for-legal-aid-rds" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=2.3"
```

Comment:
Can you please change the version to `3.0`? This addresses a naming issue.

Reply:
👍